content-repo/extra-docs/releases/20.5.0.md

```diff
@@ -0,0 +1,373 @@
+- __What were the areas where the CIRT teams were effective?__
+What were the areas where the CIRT teams were effective?.
+- __What corrective actions can prevent similar incidents in the future?__
+What corrective actions can prevent similar incidents in the future?.
+- __Suggestions and discussion of how to improve the team__
+Share ideas and information in order to improve team effectiveness in future incidents.
+- __What additional tools or resources are needed to detect, analyze, and mitigate future incidents?__
+What additional tools or resources are needed to detect, analyze, and mitigate future incidents?.
+- __How well did staff and management perform in dealing with the incident? Were the documented procedures followed? Were they adequate?__
+How well did staff and management perform in dealing with the incident? Were the documented procedures followed? Were they adequate?.
+- __What information was needed sooner?__
+What information was needed sooner?.
+- __What was the scope of the incident?__
+What was the scope of the incident?.
+- __When was the problem was first detected and by whom?__
+When was the problem was first detected and by whom?.
+- __How was the incident contained and eradicated?__
+How was the incident contained and eradicated?.
+- __Were any steps or actions taken that might have inhibited the recovery?__
+Were any steps or actions taken that might have inhibited the recovery?.
+- __What was the work performed during recovery?__
+What was the work performed during recovery?.
+- __What precursors or indicators should be watched for in the future to detect similar incidents?__
+What precursors or indicators should be watched for in the future to detect similar incidents?.
+- __Exactly what happened, and at what times?__
+Exactly what happened, and at what times?.
+- __How could information sharing with other organizations have been improved?__
+How could information sharing with other organizations have been improved?.
+- __What would the staff and management do differently the next time a similar incident occurs?__
+What would the staff and management do differently the next time a similar incident occurs?.
+
+---
+### Layouts
+
+####  7 New Layouts
+- __Pentera Insight - Summary__
+- __Review Indicators Manually For Whitelisting - Summary__
+- __SANS - Summary__
+- __NIST - Summary__
+- __MITRE ATT&CK - Indicator Details__
+- __Employee Health Check - New/Edit__
+- __Employee - Indicator Details__
+
+####  3 Improved Layouts
+- __Indicator Feed - New/Edit__
+Added edit layout for the **Indicator Feed** incident type.
+- __GDPR Data Breach - Summary__
+Changed "Date breach information" section from rows to cards.
+- __GCP Compute Engine Misconfiguration - Summary__
+New layout for 'GCP Compute Engine Misconfiguration'.
+
+
+---
+### Reputations
+  - The **CVE CVSS Score** indicator type is only available in Demisto version 4.5.x and earlier.
```

Comment:
as @gshriki asked, lets check what's this about, it looks weird

Reply:
it's from 3 months ago. Probably upon the refactor. deleting